spec.html

```diff
@@ -20785,6 +21650,25 @@ <h1>Runtime Semantics: Evaluation</h1>
         1. Set _value_.[[SourceText]] to the source text matched by |ClassExpression|.
         1. Return _value_.
       </emu-alg>
+      <emu-grammar>ClassElementName : PropertyName</emu-grammar>
+      <emu-alg>
+        1. Return the result of evaluating |PropertyName|.
```

Comment:
This isn't necessary, by the [chain production rule](https://tc39.es/ecma262/multipage/notational-conventions.html#sec-algorithm-conventions-syntax-directed-operations):

> Unless explicitly specified otherwise, all chain productions have an implicit definition for every operation that might be applied to that production's left-hand side nonterminal. The implicit definition simply reapplies the same operation with the same parameters, if any, to the chain production's sole right-hand side nonterminal and then returns the result.